flask_resty/view.py

```diff
@@ -209,11 +212,47 @@ def session(self):
 
     @settable_property
     def query(self):
+        """The SQLAlchemy query for the view.
+
+        Override this to customize the query to fetch items in this view.
+
+        By default, this applies the filter from the view's `authorization` and
+        the query options from `base_query_options` and `query_options`.
+        """
         query = self.model.query
         query = self.authorization.filter_query(query, self)
+        query = query.options(
+            itertools.chain(self.base_query_options, self.query_options),
+        )
 
         return query
 
+    #: Base query options to apply before `query_options`.
+    #:
+    #: Set this on a base class to define base query options for its
+    #: subclasses, while still allowing those subclasses to define their own
+    #: additional query options via `query_options`.
+    #:
+    #: For example, set this to ``(raiseload('*', sql_only=True),)`` to prevent
+    #: all implicit SQL-emitting relationship loading, and force all
+    #: relationship loading to be explicitly defined via `query_options`.
+    base_query_options = ()
```

Comment:
is this actually needed couldn't you just refine query? would make for a cleaner api I guess

Reply:
i think this works better naming-wise. there are also a few variations – like raise, raise_on_sql, or... whatever

i think it'd be best to leave this as is for now, but revise later as we understand things better